client/components/email-verification/index.js

```diff
@@ -8,6 +9,7 @@ import { successNotice } from 'calypso/state/notices/actions';
 
 export default function emailVerification( context, next ) {
```

Comment:
This middleware handles the success notice for email verification. We have extended it to handle new email change confirmation as well.